docs/index.md

```diff
@@ -0,0 +1,162 @@
+![IEKit](images/iekit.svg){ align=left }
+The Impact Evaluator Kit (IEKit) is an enhanced version of the [CoordinationKit](https://docs.autonolas.network/product/coordinationkit/), which leverages [Ceramic streams](https://developers.ceramic.network/docs/advanced/standards/stream-programs/) to automate the tracking and rewarding of users' contributions on the ecosystem. The Autonolas Impact Evaluator (IE) is an agent service based on the IEKit. This service is designed to track contributions in the form of Twitter mentions of the Autonolas DAO ([@autonolas](https://twitter.com/autonolas)). The service implements three main features:
+
+1. **Monitor for new users' registrations.** Any user can register as a contributor to the DAO by minting an NFT (through the [associated contract](https://etherscan.io/address/0x02c26437b292d86c5f4f21bbcce0771948274f84)) and sending a Tweet linking their wallet address.
+
+2. **Monitor for users' contributions.** The service periodically scans for new mentions of @autonolas on Twitter. If the mentions come from registered contributing users, the service increments and update their score.
```

Comment:
Actually, the scores are incremented for every user, but only those who minted and therefore have an associated ETH address will see their badges updated.

Reply:
fixed.